client/components/woopay/save-user/checkout-page-save-user.js

```diff
@@ -291,7 +293,7 @@ const CheckoutPageSaveUser = ( { isBlocksCheckout } ) => {
 						<input
 							type="hidden"
 							name="woopay_source_url"
-							value={ window.location.href }
+							value={ window.location.href.split( '?' )[ 0 ] }
```

Comment:
Same as above.

Reply:
Updated in https://github.com/Automattic/woocommerce-payments/commit/abbd2a4b3af8844c0762a3c4406cae6054540940